package.json

```diff
@@ -21,7 +21,7 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^8.7.0",
+    "@theforeman/vendor": "^8.8.0",
```

Comment:
Other `@theforeman` packages depended on `@theforeman/vendor-core`
so we should bump other packages and keep the version the same